awx/main/models/credential/__init__.py

```diff
@@ -481,6 +498,7 @@ class Meta:
         ('scm', _('Source Control')),
         ('cloud', _('Cloud')),
         ('insights', _('Insights')),
+        ('credential_management_service', _('Credential Management Service')),
```

Comment:
This key is a _liiiiiiittle_ wordy.  What about something like `external`?

Reply:
👍 I like it